docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -3,7 +3,7 @@ product: date-pickers
 title: Date picker, Time picker React components
 components: DatePicker,DateTimePicker,TimePicker
 githubLabel: 'component: DatePicker'
```

Comment:
Nitpick, but a `component: pickers` would probably be a better label for such page.
We use it as a common pickers label.

In any case, none of these values are actually used in any way as of now.
We are missing the following section from these `md` files 🤷 

```md
{{"component": "modules/components/ComponentLinkHeader.js"}}
```

Reply:
@LukasTy Agree, 👍 for a quick follow-up to match with https://next.mui.com/x/react-data-grid/.